config/src/config/state_sync_config.rs

```diff
@@ -15,6 +19,10 @@ const MAX_STATE_CHUNK_SIZE: u64 = 4000;
 const MAX_TRANSACTION_CHUNK_SIZE: u64 = 2000;
 const MAX_TRANSACTION_OUTPUT_CHUNK_SIZE: u64 = 1000;
 
+// The maximum number of concurrent requests to send
+const MAX_CONCURRENT_REQUESTS: u64 = 6;
```

Comment:
copy-pasteo? -> 3?

Reply:
Aah, no -- I actually wanted to bump it to `6` (from `3`), to match the state value prefetcher. 😄 We used to have it overridden to `6` in our terraforms, so I'm just cleaning up as I go along 🐒